Documentation/specs/runtime-configuration-file.md

```diff
@@ -128,9 +123,13 @@ This section is copied verbatim from an identical section in the input `project.
 
 These settings are read by `corehost` to determine how to initialize the runtime. All versions of `corehost` **must ignore** settings in this section that they do not understand (thus allowing new settings to be added in later versions).
 
+### `targetFramework` Section (`.config.json`)
+
+This section indicates the name and version of the shared framework in use. Where this data comes from is TBD right now, as is the name of this section. If not present, default values of `NetCoreApp` for `name` and `1.0.0` for `version` should be assumed (REVIEW: Is this the right behavior? Should we error out if it isn't present?).
```

Comment:
I dont believe we want to start with any defaults and fail to make errors obvious. This would be in line with app specifying a version and host not finding it and failing.